packages/xo-web/src/common/intl/messages.js

```diff
@@ -14,6 +14,7 @@ const messages = {
   errorUnknownItem: 'Unknown {type}',
   generateNewMacAddress: 'Generate new MAC addresses',
   memoryFree: '{memoryFree} RAM free',
+  notDefined: 'Not defined',
```

Comment:
all the message keys used in this PR are prefixed by `authToken` I think this one should also be prefixed 
```suggestion
  authTokenNotDefined: 'Not defined',
```

Reply:
I choose to not prefix this message by `authtoken` because it's a generic message. It can have other use case.